products/views.py

```diff
@@ -0,0 +1,76 @@
+from .models import Look
+from attributes.models import LookImage, Color, Texture
+import json
+from django.views import View
+from django.http import JsonResponse
```

Comment:
```suggestion
import json

from django.views import View
from django.http import JsonResponse

from .models import Look
from attributes.models import (
    LookImage, 
    Color, 
    Texture
)
```

모듈을 불러오는 순서는
1. python 기본 모듈
2. django 내부 모듈
3. 우리가 생성한 모듈입니다. 👍